packages/wonder-blocks-form/components/choice.js

```diff
@@ -19,7 +19,7 @@ type Props = {|
     value: string,
 
     /** User-defined. Whether this choice option is disabled. Default false. */
-    disabled?: boolean,
```

Comment:
I dropped the `?` because this and `checked` both have default props.

Reply:
Good catch!